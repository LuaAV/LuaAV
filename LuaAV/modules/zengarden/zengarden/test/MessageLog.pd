#N canvas 675 197 450 300 10;
#X obj 104 17 loadbang;
#X obj 104 183 print;
#X obj 173 17 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X msg 180 99 -2 3 5;
#X msg 104 99 3.7;
#X obj 104 148 log;
#X msg 139 99 0;
#X obj 104 55 t b b b;
#X connect 0 0 7 0;
#X connect 2 0 7 0;
#X connect 3 0 5 0;
#X connect 4 0 5 0;
#X connect 5 0 1 0;
#X connect 6 0 5 0;
#X connect 7 0 4 0;
#X connect 7 1 6 0;
#X connect 7 2 3 0;
