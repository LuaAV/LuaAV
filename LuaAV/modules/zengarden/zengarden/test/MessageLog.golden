[@ 0.000ms] print: -1000
[@ 0.000ms] print: -1000
[@ 0.000ms] print: 1.30833
